cmd/aries-js-worker/app.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
should we make the filename more explicit about being a node app?

Reply:
isn't "app" the convention?

Reply:
https://nodejs.org/en/docs/guides/getting-started-guide/

Reply:
We are showing both browser and node.js samples aren't we?

Reply:
we should revisit once we have npm packaging. perhaps two subdirectories for the samples would make sense.